pkg/tenant/api/v1/api.go

```diff
@@ -251,14 +251,16 @@ type TenantProjectPage struct {
 }
 
 type Project struct {
-	ID          string `json:"id" uri:"id"`
-	TenantID    string `json:"tenant_id"`
-	Name        string `json:"name"`
-	Description string `json:"description,omitempty"`
-	Owner       Member `json:"owner"`
-	Status      string `json:"status"`
-	Created     string `json:"created,omitempty"`
-	Modified    string `json:"modified,omitempty"`
+	ID           string `json:"id" uri:"id"`
+	TenantID     string `json:"tenant_id"`
+	Name         string `json:"name"`
+	Description  string `json:"description,omitempty"`
+	Owner        Member `json:"owner"`
+	Status       string `json:"status"`
+	ActiveTopics uint64 `json:"active_topics"`
+	DataStorage  uint64 `json:"data_storage"`
```

Comment:
Should `data_storage` be a `StatValue`? In other words, do we need to return units to the frontend? The field is always zero for now but I'm wondering if the frontend needs to know what the units are for the list view, or we're just displaying `GB` as a hardcoded value.